### DISCUSSION TITLE
Error at run command gpt-engineer projects/my-new-project

### DISCUSSION THREAD

**Discussion by nelsoncastro** [AntonOsika/gpt-engineer#507] [General]
`INFO:openai:error_code=model_not_found error_message="The model 'gpt-4' does not exist" error_param=model error_type=invalid_request_error message='OpenAI API error received' stream_error=False
Model gpt-4 not available for provided API key. Reverting to gpt-3.5-turbo. Sign up for the GPT-4 wait list here: https://openai.com/waitlist/gpt-4-api

Traceback (most recent call last):

  File "/opt/homebrew/bin/gpt-engineer", line 8, in <module>
    sys.exit(app())
             ^^^^^

  File "/opt/homebrew/lib/python3.11/site-packages/gpt_engineer/main.py", line 64, in main
    messages = step(ai, dbs)
               ^^^^^^^^^^^^^

  File "/opt/homebrew/lib/python3.11/site-packages/gpt_engineer/steps.py", line 53, in clarify
    user_input = get_prompt(dbs)
                 ^^^^^^^^^^^^^^^

  File "/opt/homebrew/lib/python3.11/site-packages/gpt_engineer/steps.py", line 25, in get_prompt
    "prompt" in dbs.input or "main_prompt" in dbs.input
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^

AssertionError: Please put your prompt in the file `prompt` in the project directory`

**Comment by skamenan7** [comment]
I got it working like below

c:\git\gpt-engineer-stuff\gpt-engineer(main -> origin)
(GPT_Engineer) λ python gpt_engineer\main.py projects\neural-network

**Comment by designercreativo** [comment]
i  get this error but i am a free user still.. does this mean that the private keys i generate from my account are not good? do you need to pay chatgpt4 or can only those who paid before July 6th? or is the use not even guaranteed for the latter? I'm not understanding. Or can I also try gpt Eng. with my keys?

 gpt-engineer projects/my-new-projects
INFO:openai:error_code=invalid_api_key error_message='Incorrect API key provided: sk-wAQeP***************************************EoDQ. You can find your API key at https://platform.openai.com/account/api-keys.' error_param=None error_type=invalid_request_error message='OpenAI API error received' stream_error=False
